src/redux/user/userSlice.ts

```diff
@@ -2,29 +2,43 @@ import {createSlice} from '@reduxjs/toolkit'
 import {IAuthPropsState} from '../../types/storeTypes'
 
 const initialState: IAuthPropsState = {
+  firstName: '',
+  lastName: '',
   user: '',
   email: '',
-  uid: '',
+  userId: 0,
+  authUid: '',
+  salary: 0,
+  isAuth: false,
   token: '',
-  isAuth: true,
+  avatar: 'http://res.cloudinary.com/avoo/image/upload/v1652271069/v35k9wpyszjbxhyl8xn0.jpg',
```

Comment:
Why is FE setting default image for user????
It should be done from BE image

Reply:
Done